fcl_product_components/components/gne_reference_pane/gne-reference-pane.tpl.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @file
+ * Airline name and logo pane template file.
+ *
+ * Theme hook suggestions are available in the format:
+ * airline-logo-pane--CATEGORY-NAME.tpl.php
+ * e.g. airline-logo-pane--air-transporation.tpl.php.
+ *
+ * @var $airline_name string
+ *   The product airline name.
+ *
+ * @var $airline_logo array
+ *   The product airline logo renderable array.
+ */
+?>
+<?php if ($gne_reference): ?>
+<div class="<?php print $classes; ?>">
+  <?php if (isset($title)): ?>
+        <<?php print $title_tag; ?><?php print $title_attributes; ?>>
+    <?php print $title; ?>
+        </<?php print $title_tag; ?>>
+  <?php endif; ?>
+    <div class="gne-reference-pane__gne-wrapper">
+        <div class="gne-reference-pane__gne-reference">
+          <?php print render($gne_reference); ?>
+        </div>
+    </div>
+    </div>
```

Comment:
Please lint the file.

Reply:
Add linters to the respective IDE or editor you are using, that will help.